### PR TITLE
Preset auth servers in file storage if configuration has auth servers

### PR DESCRIPTION
**Pull request by a-palchikov**
Without this, if auth servers are specified in the configuration, this leads to errors like:

```
Original Error: *trace.NotFoundError open /tmp/teleport/authservers.json: no such file or directory
Stack Trace:
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/utils/storage.go:44 bar/vendor/github.com/gravitational/teleport/lib/utils.(*FileAddrStorage).GetAddresses
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/auth/tun.go:592 bar/vendor/github.com/gravitational/teleport/lib/auth.NewTunClient
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/service/service.go:139 bar/vendor/github.com/gravitational/teleport/lib/service.(*TeleportProcess).connectToAuthService
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/service/service.go:501 bar/vendor/github.com/gravitational/teleport/lib/service.(*TeleportProcess).RegisterWithAuthServer.func1
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/service/supervisor.go:250 bar/vendor/github.com/gravitational/teleport/lib/service.ServiceFunc.Serve
    /home/foo/bar/vendor/github.com/gravitational/teleport/lib/service/supervisor.go:144 bar/vendor/github.com/gravitational/teleport/lib/service.(*LocalSupervisor).serve.func2
    /home/foo/dev/go/go/src/runtime/asm_amd64.s:2087 runtime.goexit
User Message: open /tmp/teleport/authservers.json: no such file or directory
  file=auth/tun.go:577 func=auth.NewTunClient
```

### CODE REVIEW

**Comment by sofuture** [comment]
ok to test

**Comment by a-palchikov** [comment]
ok to test

File: lib/service/service.go [hunk 1]
```diff
@@ -135,6 +135,12 @@ func (process *TeleportProcess) connectToAuthService(role teleport.Role) (*Conne
 	storage := utils.NewFileAddrStorage(
 		filepath.Join(process.Config.DataDir, "authservers.json"))
 
+	if len(process.Config.AuthServers) > 0 {
```
Comment (klizhentas) [inline lib/service/service.go]: This will overwrite the previous value if it exists, so we need `SetIfNotExists` semantics

**Review by klizhentas — CHANGES_REQUESTED** [review]

**Comment by kontsevoy** [comment]
I don't get this comment:

> Without this, if auth servers are specified in the configuration, this leads to errors like...

In my case I _do_ have auth servers specified in the config. yet no errors like this get reported. @a-palchikov , can you ping me on Slack about this plz?

**Comment by kontsevoy** [comment]
Update: just had a chat with `a-palchikov`. The logic in `master` is totally fine: the auth server cache is not supposed to exist here anyway. The "problem" is simply verbose logging, lets lower the severity of the log message.

**Pull request merged**